log10/feedback/autofeedback.py

```diff
@@ -119,3 +159,14 @@ def auto_feedback_icl(task_id: str, content: str, file: str, completion_id: str,
             content = f.read()
     results = auto_feedback_icl.predict(text=content)
     console.print_json(results)
+
+
+@click.command()
+@click.option("--completion-id", required=True, help="Completion ID")
+def get_autofeedback_cli(completion_id: str):
```

Comment:
how about name it `get_autofeedback`? not sure we want the `cli` in the function name.